### ISSUE TITLE
Warning formats is deprecated and will be removed from Rails 6.1

### ISSUE THREAD

**Issue by kaluznyo** [rspec/rspec-rails#2094]
When I run spec with rails 6.0.0.beta2 and rspec master I got 
```
DEPRECATION WARNING: formats is deprecated and will be removed from Rails 6.1 (called from block in nullify_template_rendering at /Users/okaluzny/.rvm/gems/ruby-2.6.1@app/bundler/gems/rspec-rails-7720e7b1c8c6/lib/rspec/rails/view_rendering.rb:59
```

Ruby version: 2.6.1
Rails version: 6.0.0.beta2
Rspec version: RSpec 3.9.0.pre (master)

**Comment by JonRowe** [comment]
Please note `4-0-dev` is where Rails 6 fixes will end up.

**Comment by pirj** [comment]
@kaluznyo This is [fixed in `rspec-rails` 4.0.0beta](https://github.com/rspec/rspec-rails/blob/bd54ec4f9c63d9e36dbb3196a59b0908561e287d/lib/rspec/rails/view_rendering.rb#L65), which is to be used with Rails 6+.

Please feel free to reopen if the warning pops up with 4.0.0 or or `master`.
